CONTRIBUTING.MD

```diff
@@ -61,7 +58,6 @@ Before submitting your pull request for code review, please ensure your code has
   - Java: Please follow our [testing best practices].
   - Have a look at other tests in the same module for how it works.
   - In rare cases, it is not feasible to write an automated test. Please ask us if you think that is the case for your contribution.
-- Do your commits follow the [Conventional Commits] guidelines and syntax?
```

Comment:
We don't really follow this strictly. Much of this doc came from a different team's guide....I'm removing this, because not even I follow it.